### PR TITLE
Always translate the log level

### PR DESCRIPTION
**Pull request by mariocasciaro**
Fixes the error "Invalid log level.  Using the default instead." produced when specifying the 'verbose' or 'http' log levels. Also add the 'debug' level mapping.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,19 +29,20 @@ module.exports = class LogDNATransport extends Transport {
...
     if (!info.message) {
       // Send the incoming object payload as the message
-      const level = levelTranslate.get(info.level)
       this.logger.log(info, level)
       callback(null, true)
       return
     }
 
-    const {level, message, indexMeta, timestamp, ...meta} = info
+    const {level: ignored, message, indexMeta, timestamp, ...meta} = info
```
Comment (darinspivey) [inline index.js:45]: For clarity, we'd prefer the variable name to be `_` rather than `ignored`, since "ignored" could still some sort of functionality.

**Review by darinspivey — CHANGES_REQUESTED** [review]
Appreciate the corrections here!

**Comment by darinspivey** [comment]
@logdnabot please test this

File: index.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ const levelTranslate = new Map([
 , ['info', 'info']
 , ['http', 'debug']
 , ['verbose', 'debug']
+, ['debug', 'debug']
```
Comment (darinspivey) [inline index.js:14]: The linter is failing with trailing spaces on this line.  Please run `npm run test` prior to re-pushing anything.

Comment (darinspivey) [reply index.js:14]: You can also run `npm run lint -- --fix` to auto-correct those issues

**Comment by darinspivey** [comment]
If you rebase this onto origin, any lint or test errors will then show up in github checks

**Comment by mariocasciaro** [comment]
Done

**Comment by darinspivey** [comment]
@logdnabot please test this

**Review by darinspivey — APPROVED** [review]
Looks good, thanks for this update.

**Review by jorgebay — APPROVED** [review]
Thanks for the fix @mariocasciaro !

**Comment by mariocasciaro** [comment]
👍 thank you

**Pull request merged**